src/Helper/Notifier.php

```diff
@@ -20,6 +20,10 @@ public function __construct(
 
     public function send(string $message, ?string $title = null): void
     {
+        if (false === context()->notify) {
```

Comment:
We should not use userland function in all service. Inject what you need instead (the context registry)